cmd/podman/create_cli.go

```diff
@@ -98,6 +98,24 @@ func parseVolumes(volumes []string) error {
 	return nil
 }
 
+func parseVolumesFrom(volumesFrom []string) error {
+	if len(volumesFrom) == 0 {
+		return nil
```

Comment:
Do you need this guard?  Without it, I expect the following loop to be a no-op, so we'd hit the `nil` return at the end of the function.

Reply:
Right, not needed. Removed it.